impls/monero.dart/lib/src/monero.dart

```diff
@@ -1357,14 +1350,16 @@ class MoneroWallet implements Wallet2Wallet {
     return monero.Wallet_setDevicePin(walletPtr, passphrase: passphrase);
   }
   
-  @override
-  void setDeviceReceivedData(Pointer<UnsignedChar> data, int len) {
-    monero.Wallet_setDeviceReceivedData(walletPtr, data, len);
+  static void setDeviceReceivedData(Pointer<UnsignedChar> data, int len) {
+    monero.Wallet_setDeviceReceivedData(data, len);
   }
   
-  @override
-  void setDeviceSendData(Pointer<UnsignedChar> data, int len) {
-    monero.Wallet_setDeviceSendData(walletPtr, data, len);
+  static void setDeviceSendData(Pointer<UnsignedChar> data, int len) {
+    monero.Wallet_setDeviceSendData(data, len);
+  }
+
+  static void serLedgerCallback(Pointer<NativeFunction<Void Function(Pointer<UnsignedChar>, UnsignedInt)>> callback) {
```

Comment:
```suggestion
  static void setLedgerCallback(Pointer<NativeFunction<Void Function(Pointer<UnsignedChar>, UnsignedInt)>> callback) {
```

Reply:
you didn't just leave review the moment I merged it